storage/time-series/influxdb/retriever/LICENSE-2.0.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
I used this file name LICENSE-2.0.txt, but it would be better to use LICENSE to be recognized by Github.